OlinkAnalyze/data-raw/tutorial_figures.R

```diff
@@ -0,0 +1,150 @@
+# Tutorial Figures --------------------------------------------------------
```

Comment:
Figure sizes are too small (likely due to small viewer when using ggsave) -> regenerate and specify sizes.